sops/__init__.py

```diff
@@ -620,6 +632,7 @@ def walk_and_decrypt(branch, key, aad=b'', stash=None, digest=None,
     for k, v in branch.items():
         if k == 'sops' and isRoot:
             continue    # everything under the `sops` key stays in clear
+        v_unencrypted = unencrypted or k.endswith(unencrypted_suffix)
```

Comment:
We used the same language (i.e. `unencrypted`) for consistency to make the connection obvious to programmers.

Additionally, we need to pass `v_unencrypted` (or whatever we call it) to nested branch traversals since they need to not encrypt themselves as well (e.g. `{'a_unencrypted': {'b': 'c'}}`). As a result, performing boolean flipping will probably cause more confusion than help =/